ui/packages/catalog-ui-search/src/main/webapp/js/model/Theme.js

```diff
@@ -34,7 +34,8 @@ var colorVariables = [
     'customBackgroundModal',
     'customBackgroundSlideout'
 ];
-var themeableVariables = spacingVariables.concat(colorVariables);
+const transitionVariables = ['coreTransitionTime'];
+const themeableVariables = spacingVariables.concat(colorVariables).concat(transitionVariables);
```

Comment:
❓ Was this a partial attempt at making the transition time with alpha changes configurable?

Reply:
Yeah, an earlier version had some animation and I needed the transition time.  It can be removed.